legend-sdlc-generation-service/pom.xml

```diff
@@ -188,11 +188,6 @@
             <artifactId>jetty-servlet</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
does this cause Whitesource problem?

Reply:
I removed the dependency because it's no longer explicitly used. Nothing to do with Whitesource.